### PR TITLE
cohort generation process and status is now visible to another users

### LINKED ISSUE #2169: Cohort generation does not pick up new generation status with security enabled
### Expected behavior

When a cohort generation finishes, the polling mechanism finds the updated status and shows as "complete". 

### Actual behavior

When generating a cohort with security enabled, the polling mechanism uses the "notifications" area which only shows jobs for that user. If a given cohort is being generated by another user, the current user will not see the updated status in the UI and potentially kick off another job to generate the same cohort.

### Steps to reproduce behavior

- User A kicks off a generation on a cohort on a data source
- User B opens the same cohort and reviews the generations
- User B does not see that the generation created by User A is in progress

Related to #2168

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
fixes #2169

### CODE REVIEW

**Comment by wivern** [comment]
@ssuvorov-fls And I'd like to remind you not to forget resolve conflicts before fixing

File: js/pages/cohort-definitions/cohort-definition-manager.js [hunk 1]
```diff
@@ -437,56 +437,54 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
...
 							})[0];
 
 							if (source) {
-								// only bother updating those sources that we know are running
-									if (this.isSourceRunning(source)) {
-									source.status(info.status);
-									source.includeFeatures(info.includeFeatures);
-									source.isValid(info.isValid);
-									source.startTime(momentApi.formatDateTime(new Date(info.startTime)));
+								source.status(info.status);
```
[the remark this answers is not in the document]
Comment (ssuvorov-fls) [reply js/pages/cohort-definitions/cohort-definition-manager.js:465]: No, we have to show statuses of generations started by other users, so we can't say whether source is running or not and have to change the whole list each time

File: js/pages/cohort-definitions/cohort-definition-manager.js [hunk 2]
```diff
@@ -437,56 +437,54 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 			this.stopping = ko.pureComputed(() => this.cohortDefinitionSourceInfo().reduce((acc, target) => ({...acc, [target.sourceKey]: ko.observable(false)}), {}));
 			this.isSourceStopping = (source) => this.stopping()[source.sourceKey];
 
-			this.pollForInfo = () => {
+			this.pollForInfoPeriodically = () => {
```
[the remark this answers is not in the document]
Comment (ssuvorov-fls) [reply js/pages/cohort-definitions/cohort-definition-manager.js:440]: done

File: js/pages/cohort-definitions/cohort-definition-manager.js [hunk 3]
```diff
@@ -801,6 +800,18 @@ define(['jquery', 'knockout', 'text!./cohort-definition-manager.html',
 				}
 			}
 
+			isCancelDisabled(source) {
+				return this.isSourceStopping(source)() || this.processedByAnotherUser(source);
+			}
+
+			processedByAnotherUser(source) {
```
[the remark this answers is not in the document]
Comment (ssuvorov-fls) [reply js/pages/cohort-definitions/cohort-definition-manager.js:807]: done

**Review by anton-abushkevich — APPROVED** [review]

**Review by wivern — APPROVED** [review]

**Pull request merged**